src/report.ts

```diff
@@ -18,11 +18,17 @@ const sleep = (duration: number) => new Promise(resolve => setTimeout(resolve, d
 async function pollReport(
   options: UploadReportOptions,
 ): Promise<Result<AnalysisFailedResponse | ReportResult, GetAnalysisErrorCodes>> {
-  // Return early if project name is not provided.
   const projectName = options.report?.projectName?.trim();
+  const projectNameMaxLength = 64;
   if (!projectName || projectName.length === 0) {
     throw new Error('"project-name" must be provided for "report"');
   }
+  if (projectName.length > projectNameMaxLength) {
+    throw new Error(`project-name "${projectName}" must not exceed ${projectNameMaxLength} characters`);
```

Comment:
`nit`: would keep the messages consistent with the above, i.e.
```suggestion
    throw new Error(`"project-name" must not exceed ${projectNameMaxLength} characters`);
```